### ISSUE TITLE
Visual Studio Build Errors

### ISSUE THREAD

**Issue by GloveNinja217** [Microsoft/AzureUsageAndBillingPortal#58]
I have deployed all the resources into Azure and have edited the AUI Solution file in Visual Studio. When I try to Rebuild the solution, I get a bunch of errors and warnings.

I am by no means a Developer so I'm not sure what to do about these issues. I'm wondering if anyone else has seen these and has an idea about how to fix it.

Also, I came across issues where the Storage Account key wouldn't appear in the output and even the Client ID wouldn't come up either (I'm assuming that's just the subscription ID).

![aubp](https://cloud.githubusercontent.com/assets/14881605/22553324/49bb8772-e92a-11e6-9e9b-daf5e33131d1.png)

**Comment by GloveNinja217** [comment]
I noticed in the video here: https://channel9.msdn.com/blogs/Mustafa-Kasap/How-to-Setup-the-Azure-Usage--Billing-Portal#time=05m20s, at about the 10:00 mark, there was something called the NuGet Package Manager. So I checked into that in my Visual Studio 2015 program and before I rebuilt the solution, I updated everything. I kept hitting Yes-to-All and I was very hopeful. However, when I compiled the code this time, I got different errors than before.

Is there some kind of configuration in my Visual Studio that I'm missing? Or am I running incorrect versions of things?

![aubp2](https://cloud.githubusercontent.com/assets/14881605/22557422/8142b02c-e938-11e6-8c03-b51568e2d21f.png)
